src/Microsoft.AspNet.OData.Shared/EdmODataAPIHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="EdmODataAPIHandler.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.OData
+{
+
+    /// <summary>
+    /// Handler Class to handle users methods for create, delete and update.
+    /// This is the handler for data modification where there is no CLR type.
+    /// </summary>
+    public abstract class EdmODataAPIHandler
```

Comment:
The comments say that this class contains methods for create, delete and update. But I don't see any method for performing updates defined in the class.